apps/storybook/stories/FeaturedQuoteBlock.stories.tsx

```diff
@@ -1,26 +1,23 @@
+import FeaturedQuoteBlock from '@evans/pages/friend/FeaturedQuoteBlock';
 import type { Meta, StoryObj } from '@storybook/react';
-import FeaturedQuoteBlock from '../../next-evans/components/pages/friend/FeaturedQuoteBlock';
-import '../styles/globals.css';
+import { LONG_LOREM, SHORT_LOREM, props } from './helpers';
 
-const meta: Meta<typeof FeaturedQuoteBlock> = {
-  title: 'Friend/FeaturedQuoteBlock',
+const meta = {
+  title: 'Friend/FeaturedQuoteBlock', // eslint-disable-line
   component: FeaturedQuoteBlock,
-};
+  parameters: { layout: `fullscreen` },
+} satisfies Meta<typeof FeaturedQuoteBlock>;
 
-export default meta;
+type Story = StoryObj<typeof meta>;
 
-type Story = StoryObj<typeof FeaturedQuoteBlock>;
+export const Default: Story = props({
+  quote: SHORT_LOREM,
+  cite: `John Doe (on his death bed)`,
+});
 
-export const Default: Story = {
-  args: {
-    quote: `Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptas tempora necessitatibus magni culpa ab placeat saepe vitae doloremque ipsa debitis laudantium, eos quasi veniam, dicta alias. Aliquam reiciendis aliquid perspiciatis? Lorem ipsum, dolor sit amet consectetur adipisicing elit.`,
-    cite: `John Doe (on his death bed)`,
-  },
-};
+export const Long: Story = props({
```

Comment:
this little `props` helper i cooked up just prevents us from having to nest our props inside of `args` everywhere, and reduces a level of indentation and 2 lines for every single story.